recipes/libjwt/all/conandata.yml

```diff
@@ -1,3 +1,9 @@
 sources:
-  "1.18.3":
-    url: "https://github.com/benmcollins/libjwt/archive/refs/tags/v1.18.3.zip"
+  "3.2.0":
```

Comment:
@AbrilRBS thanks for the suggestion, but as mentioned in https://github.com/conan-io/conan-center-index/pull/26459 we actually need the v1 of this library in order to build [libstirshaken](https://github.com/signalwire/libstirshaken). I have nothing against adding v3 as long as v1 is also available.